pkg/modifycontroller/controller_test.go

```diff
@@ -346,26 +401,10 @@ func setupFakeK8sEnvironment(t *testing.T, client *csi.MockClient, initialObject
 		workqueue.DefaultTypedControllerRateLimiter[string]())
 
 	/* Start informers and ModifyController*/
-	stopCh := make(chan struct{})
-	informerFactory.Start(stopCh)
-
-	go controller.Run(1, t.Context())
-
-	/* Add initial objects to informer caches */
```

Comment:
We don't need to add them manually, as informer will read them from `kubeClient`. `ctrl.init(ctx)` will wait for cache sync.